### PR TITLE
Fix `cupyx.scipy.sparse._base` tests for SciPy 1.11

### PR DESCRIPTION
**Pull request by asi1024**
Part of #7664.
https://docs.scipy.org/doc/scipy/release/1.11.0-notes.html#scipy-sparse-improvements

### CODE REVIEW

File: tests/cupyx_tests/scipy_tests/sparse_tests/test_base.py [hunk 1]
```diff
@@ -81,4 +82,4 @@ def test_asformat_none(self, xp, sp):
     @testing.numpy_cupy_equal(sp_name='sp')
     def test_maxprint(self, xp, sp):
         s = self.dummy_class(sp)(maxprint=30)
-        return s.getmaxprint()
+        return s.maxprint
```
Comment (asi1024) [inline tests/cupyx_tests/scipy_tests/sparse_tests/test_base.py:85]: note: `getmaxprint` was deprecated in SciPy 1.11.

Comment (kmaehashi) [reply tests/cupyx_tests/scipy_tests/sparse_tests/test_base.py:85]: Note for later reference: this is deprecated in sparray but not in spmatrix, so we don't have to add DeprecationWarning for getmaxprint.

https://docs.scipy.org/doc/scipy/release/1.11.0-notes.html#deprecated-features

**Comment by kmaehashi** [comment]
LGTM! Remaining head test failures will be fixed in separate PRs.

**Pull request merged**